app/assets/stylesheets/omni_nav/omni_nav_v2/omni-nav.scss

```diff
@@ -106,6 +117,8 @@ html.omni-nav-v2 {
       background-position: center;
       background-repeat: no-repeat;
       text-indent: -9999px;
+      border-left: none;
```

Comment:
This rule isn't doing anything, you already have `border: none` on line 110 and you have the same rule on line 383 under a more specific selector so that one overwrites this one.